src/test/kotlin/eli/inevitable/enumerations/AlignTest.kt

```diff
@@ -1,8 +1,61 @@
 package eli.inevitable.enumerations
 
+import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.TestInstance
+import kotlin.test.assertEquals
 
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
Is this only for simplified kotlin integration? https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/TestInstance.html

Reply:
I'm just using it to help define the scope of the test. I'm not sure if there's another way of handling this for Kotlin tests. Most of my tests are dead simple, so we don't need a lot of complexity in the testing framework.